learns-app-content/week-08/objectives.md

```diff
@@ -1,23 +1,19 @@
 <!-- h1, h2 already used by CTD Learns -->
 ### Topics Covered
 
-- topic 1
-- topic 2
-- topic n
+- useMemo
+- useCallback
 
 ### Lesson Objectives
 
-#### Objective 1: Topic 1
+By the end of this lesson, we will:
 
-<!-- - {{blooms action verb}} sub-topic takeaway -->
-<!-- - {{blooms action verb}} sub-topic takeaway -->
-
-#### Objective 2: Topic 2
+#### Objective 1: useMemo
 
 <!-- - {{blooms action verb}} sub-topic takeaway -->
 <!-- - {{blooms action verb}} sub-topic takeaway -->
```

Comment:
- demonstrate how useMemo can be used to optimize an app's performance
- demonstrate how useCallback can be used to optimize an app's performance